changelog

```diff
@@ -1,3 +1,174 @@
+Version 0.10.11
+  * Big changes:
+    * Added a new alien species in a group of systems that is difficult to navigate. (@Azure3141, @beccabunny)
+    * Systems and planets can now have display names. (@TomGoodIdea)
+    * Added a new map mode that draws star graphics instead of rings. (@Azure3141)
+    * Added individual volume controls for different kinds of sounds. (@tibetiroka)
```

Comment:
```suggestion
    * Added a new map mode that draws star graphics instead of rings. (@Azure3141)
    * Added individual volume controls for different kinds of sounds. (@tibetiroka)
    * Systems and planets can now have display names. (@TomGoodIdea)
```